keras/metrics/metrics.py

```diff
@@ -3194,6 +3194,10 @@ def accuracy(y_true, y_pred):
 def binary_accuracy(y_true, y_pred, threshold=0.5):
   """Calculates how often predictions match binary labels.
 
+  Note: calls metrics_utils.binary_matches with mean reduction. This maintains
+  public facing binary_accuracy behavior and seperates it from the vital
+  behavior of the binary_matches method needed in backend dependencies.
```

Comment:
This note is related to internal implementation details and should not appear in the docstring, which is the public-facing documentation of the function.

Reply:
Noted will fix, my bad.